### PR TITLE
[python] Add MultiscaleImage level SpatialData exporter

### PR DESCRIPTION
**Pull request by jp-dark**
Exports a single resolution level of `MultiscaleImage` to a SpatialData `Image2DModel` or `Image3DModel`.

### CODE REVIEW

File: apis/python/src/tiledbsoma/experimental/outgest.py [hunk 1]
```diff
@@ -144,3 +146,73 @@ def to_spatial_data_shapes(
...
+        )
+    new_axis_names, image_dim_map = _convert_axis_names(
+        orig_axis_names, image.data_axis_order
+    )
+
+    # Get the URI of the requested level.
+    if level is None:
+        if image.level_count != 1:
+            raise ValueError(
+                "Parmeter ``level`` is required for a multiscale image with more than "
```
Comment (johnkerl) [inline apis/python/src/tiledbsoma/experimental/outgest.py:182]: Are double-backticks the right thing here?

Comment (johnkerl) [reply apis/python/src/tiledbsoma/experimental/outgest.py:182]: Also `Parmeter` -> `Parameter`

**Review by johnkerl — APPROVED** [review]
LGTM (with nits) -- but I defer to @ivirshup for domain expertise

**Comment by jp-dark** [comment]
[sc-57969](https://app.shortcut.com/tiledb-inc/story/57969/exporter-for-multiscaleimage-with-one-level-to-spatialdata-compatible-xarry-dataarray)

**Pull request merged**